lib/pbench/agent/tool_data_sink.py

```diff
@@ -1131,11 +1505,17 @@ def main(argv):
         logger.error("External 'tar' executable not found")
         return 2
 
-    if not os.environ.get("_PBENCH_UNIT_TESTS") and not find_executable("podman"):
-        logger.error(
-            "Podman is not installed on this system (required by some tools, aborting launch)"
-        )
-        return 3
+    global cp_path
+    cp_path = find_executable("cp")
+    if cp_path is None:
+        logger.error("External 'cp' executable not found")
+        return 2
+
+    global podman_path
+    podman_path = find_executable("podman")
```

Comment:
Ugh; I hate globals. Couldn't this be part of an object state?

Reply:
I'll be puling the removal of podman commit from PR #2076 which I believe addresses this.